src/pages/summer/Login/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import './LoginInput.scss';
+
+class LoginInput extends React.Component {
+  state = {
+    idInput: '',
+    pwInput: '',
+    idVld: false,
+    pwVld: false,
+  };
+
+  gotoMain = e => {
+    const { IP_ADDRESS } = this.props;
+    e.preventDefault();
```

Comment:
```suggestion
  gotoMain = e => {
    const { idInput: email, pwInput: password } = this.state
    e.preventDefault();
```
구조분해 할당을 해주면서 다른 변수명으로 별칭을 지어줄 수 있습니다